qa/rpc-tests/test_framework/mininode.py

```diff
@@ -39,13 +39,24 @@
     zcash_person,
 )
 
-OVERWINTER_PROTO_VERSION = 170003
 BIP0031_VERSION = 60000
 SPROUT_PROTO_VERSION = 170002  # past bip-31 for ping/pong
+OVERWINTER_PROTO_VERSION = 170003
 SAPLING_PROTO_VERSION = 170006
+BLOSSOM_TESTNET_PROTO_VERSION = 170007
```

Comment:
Remove this constant:
- `17007` was Sapling mainnet (and testnet because of the testnet change). `17008` is the correct constant here for Blossom.
- The constants in this file are specifically for regtest, which is the only kind of node the RPC tests ever test against (and except for Sapling, the regtest versions match testnet), so a testnet-specific constant is unnecessary.

Reply:
```suggestion
```

Reply:
Oops, thanks. Please also add a comment saying that these are the regtest versions.